src/modules/notices/NoticesPage.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="mb-5 flex justify-between border-primary-40 border-b pb-3">
+    <PageTitle :title="t('menu.notices')"></PageTitle>
+    <div class="flex-0 ml-3">
+      <AppButton to="/notices/add">{{ t('notices.addNotice') }}</AppButton>
+    </div>
+  </div>
+  <AppTable
+    v-model:sort="currentSort"
+    :headers="headers"
+    :items="noticesTable.items"
+    class="w-full"
+  >
+    <template #status="{ value }">
+      <AppItemStatus :status="value" />
+    </template>
+    <template #name="{ item, value }">
+      <router-link
+        :to="'/notices/' + item.id"
```

Comment:
I must be missing a bit of context here – we also have `item.url`, which as far as tests are concern do resolve?

Reply:
`item.url` is the link you want the notice to point to when a user clicks it
![image](https://user-images.githubusercontent.com/2084823/158594633-684f488f-c255-4269-88ed-3c78654e5afa.png)

/notices/`item.id` is the admin page to edit the notice (currently doesn't exist). I should make this point to the legacy app's page for now, will do that now